src/DotLiquid/FileSystems/LocalFileSystem.cs

```diff
@@ -19,13 +19,16 @@ namespace DotLiquid.FileSystems
     /// </summary>
     public class LocalFileSystem : IFileSystem
     {
+        internal readonly Regex DirectorySeparators = new Regex(@"[\\/]", RegexOptions.Compiled);

```

Comment:
could this be declared just once in a common file?

Reply:
if we make it static, yes, and put in DotLiquid/Liquid.cs but then anyone not using include (or using an alternate storage) would have it load for nothing.

Reply:
Could we initialize just in time with C# Lazy?